src/PlusDataCollection/Capistrano/vtkPlusCapistranoVideoSource.cxx

```diff
@@ -1589,20 +1589,20 @@ PlusStatus vtkPlusCapistranoVideoSource::SetDepthMm(float depthMm)
 PlusStatus vtkPlusCapistranoVideoSource::UpdateUSParameters()
 {
   RETURN_WITH_FAIL_IF(this->UpdateUSProbeParameters() == PLUS_FAIL,
-      "Failed to Update US probe parameters");
+                      "Failed to Update US probe parameters");
   RETURN_WITH_FAIL_IF(this->UpdateUSBModeParameters() == PLUS_FAIL,
-      "Failed to Update US BMode parameters");
+                      "Failed to Update US BMode parameters");
   return PLUS_SUCCESS;
 }
 
 // ----------------------------------------------------------------------------
 PlusStatus vtkPlusCapistranoVideoSource::UpdateUSProbeParameters()
 {
   // Set US Probe scan mode -------------------------------------------------
-  if(this->BidirectionalMode)
-    usbSetBidirectionalMode();
+  if (this->BidirectionalMode)
+  { usbSetBidirectionalMode(); }

```

Comment:
Doesn't the style call for braces on lines by themselves to emphasize an if? This is hardly better, if not even worse than no braces.

Reply:
Yes, my astyle formatter was not set up correctly. It should place them in brackets on their own lines. I will correct.